src/components/external-events/ExternalEventTypeManagementTab.svelte

```diff
@@ -0,0 +1,21 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { externalEventTypes } from '../../stores/external-event';
+  import type { DataGridColumnDef } from '../../types/data-grid';
+  import type { ExternalEventType } from '../../types/external-event';
+  import { getRowIdExternalEventType } from '../../utilities/externalEvents';
+  import DataGrid from '../ui/DataGrid/DataGrid.svelte';
+
+  export let externalEventTypeColumnDefs: DataGridColumnDef<ExternalEventType>[] = [];
+  export let filterString: string = '';
+
+  let externalEventTypeDataGrid: DataGrid<ExternalEventType>;
+</script>
+
+<DataGrid
+  bind:this={externalEventTypeDataGrid}
+  columnDefs={externalEventTypeColumnDefs}
+  rowData={$externalEventTypes.filter(eet => eet.name.includes(filterString))}
```

Comment:
This is very pedantic, but do you mind renaming `eet` to `externalEventType`. It'll save someone a split second when reading this

Reply:
Also `DataGrid` has a `filterExpression` prop that handles filtering itself. You should be able to just pass `filterString` into that rather than manually filtering the original data.